webapp/utils/utils.jsx

```diff
@@ -1195,10 +1195,8 @@ export function canCreateCustomEmoji(user) {
         return false;
     } else if (window.mm_config.RestrictCustomEmojiCreation === 'admin') {
         // check whether the user is an admin on any of their teams
```

Comment:
This comment doesn't really make sense any more, as you appear to have changed the  the code to only check if they're an admin on the current team.